duckduckgo/Android
# Autofill autoprompt + pixel

Task/Issue URL: https://app.asana.com/0/1202552961248957/1202778915697069/f

### Description
This fires a pixel when autofill autoprompt is dismissed. **Note: This also adds to _.gitignore_ the source files for autofill and untracks them from the repo. This will improve the diffs for future autofill updates.**

### Steps to test this PR

_Pixel fired_
1. Visit a login page where you have credentials. For example, https://privacy-test-pages.glitch.me/autofill/autoprompt/1-standard-login-form.html (save credentials for this page if you haven't yet).
2. Provided the form is front and center and not covered on pageload you should be prompted to autofill your credentials automatically (this is the case on the page linked above).
3. Dismiss the prompt and look at the Xcode console. You should see a pixel called `m_autofill_logins_autoprompt_dismissed`.

_No pixel_
1. Visit a login page where you have credentials but the form does not trigger autoprompt. For example, https://privacy-test-pages.glitch.me/autofill/autoprompt/2-form-in-modal.html.
2. Dismiss the prompt and look at the Xcode console. You should not see a pixel called `m_autofill_logins_autoprompt_dismissed`.
